grammars/julia.cson

```diff
@@ -288,13 +288,17 @@ repository:
         name: "keyword.operator.applies.julia"
       }
       {
-        match: "(?:\\||\\&|~)"
+        match: "(?:\\||\\.\\||\\&|\\.\\&|~|\\.~|⊻|\\.⊻)"
         name: "keyword.operator.bitwise.julia"
       }
       {
-        match: "(?:\\+\\+|--|\\+|\\.\\+|-|\\.\\-|\\*|\\.\\*|//(?!=)|\\.//(?!=)|/|\\./|%|\\.%|\\\\|\\.\\\\|\\^|\\.\\^)"
+        match: "(?:\\+\\+|--|\\+|\\.\\+|-|\\.\\-|\\*|\\.\\*|//(?!=)|\\.//(?!=)|/|\\./|%|\\.%|\\\\|\\.\\\\|\\^|\\.\\^|÷|\\.÷|⋅|\\.⋅|∩|\\.∩|∪|\\.∪|×|√|∛)"
         name: "keyword.operator.arithmetic.julia"
       }
+      {
+        match: "(?:∘)"
+        name: "keyword.operator.compose.julia"
+      }
```

Comment:
what is the reason why this operator should be separated, rather than appended to `keyword.operator.arithmetic.julia` ?

Reply:
I did that because the compose operator can do much more than compose arithmetic function, can be used in any function, but if you think it should be in arithmetic, I'll put it in

Reply:
Should I put as arithmetic? Never got a response

Reply:
Nah, that's fine. Sorry for taking so long to look into this.